src/OpenTelemetry.Exporter.Geneva/Internal/Transports/UnixUserEventsDataTransport.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET
+
+using Microsoft.LinuxTracepoints.Provider;
+
+namespace OpenTelemetry.Exporter.Geneva.Transports;
+
+internal sealed class UnixUserEventsDataTransport : IDisposable
+{
+    public const string LogsTracepointName = "geneva_logs";  // TODO: find the correct tracepoint name
+    public const string LogsTracepointNameArgs = $"{LogsTracepointName} u32 protocol;char[8] version;__rel_loc u8[] buffer";  // TODO: find the correct tracepoint args
+    public const string EventHeaderDynamicProviderName = "MicrosoftOpenTelemetryLogs";
```

Comment:
Is there a reasoning behind the name `MicrosoftOpenTelemetryLogs`?

Reply:
I think this is independent to the agent. The user could expect to configure this provider name when using the exporter. No matter which name is used here, the agent can read it via its own configuration.

Reply:
Got it! I'll update it to take custom provider name in another PR.